apps/common/models.py

```diff
@@ -950,3 +951,23 @@ class Meta:
                 fields=["country", "product"], name="common_countryclassified_country_code_product_code_uniq"
             )
         ]
+
+
+class HeaProfile(Model):
+    """
+    A profile to store data associated with a user to be used by the Livelihoods Explorer
+    to create a dynamic user experience.
+    """
+
+    user = models.OneToOneField(User, on_delete=CASCADE, primary_key=True, unique=True)
+    expert = models.BooleanField(default=False)
```

Comment:
@Jake-Stoll do you need `expert`, `skip_tour` and `tour_last_viewed` as concrete fields in the HEA user profile?  Will we have a tour or an expert mode? If we do, would they be better implemented as part of the JSON structure?